ocs_ci/ocs/jenkins.py

```diff
@@ -349,6 +352,36 @@ def export_builds_results_to_googlesheet(
              f"test_run_name:{utils.get_testrun_name()}"], 2
         )
 
+    def get_nodes(self, node_type='worker', num_of_nodes=1):
```

Comment:
I think it will be better to use constants here: replace `node_type='worker'` with `node_type=constants.WORKER_MACHINE` or if it's a master node change to `node_type=constants.MASTER_MACHINE`